client/landing/stepper/hooks/use-podcast-title.ts

```diff
@@ -16,11 +13,17 @@ export default function usePodcastTitle(): string | null {
 		if ( ! anchorFmPodcastId ) {
 			return;
 		}
+
+		//Prevent memory leak when API has not finished fetching/component has not mounted
```

Comment:
👀 Is this something we need to take care of in other places?

Reply:
It does feel strange. I'll look into it to see if I can find a different approach

Reply:
I would agree - this seems like a better solution. 🙂

Reply:
Weird, I swear I tried that approach first but it didn't work :D I must have done something wrong. I'll give it another go!